AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -39,7 +39,61 @@ NS_ASSUME_NONNULL_BEGIN
  * variables.
  */
 
-@interface ASDisplayNode (Subclassing)
+@protocol ASInterfaceState <NSObject>
```

Comment:
Let's name this e.g. `ASInterfaceStateDelegate`. It's OK for something to be its own delegate so this term applies both to nodes and node controllers.